### DISCUSSION TITLE
How to get data results

### DISCUSSION THREAD

**Discussion by 784622315** [simnibs/simnibs#384] [2. General]
Dear Professor, Hello, I am a new student of simnibs, I have simulated transcranial magnetic stimulation on this software. There are some questions I want to ask you. I know that the effect of transcranial magnetic stimulation depends on its stimulation depth, stimulation area and so on. I want to ask you how you can obtain these data. If you can answer my question, i would be very grateful

**Comment by Kstoffer** [comment]
When you run a simulation simNIBS will report some metrics related to the focality and depth (you can find these in "fields_summary.txt"). However, how to interpret this will depend on what you intent to use the simulations for (the electric field in the desired region specifically, you will have to inspect this for example using the visualization in gmsh).
Best,
Kristoffer
